app/components/courses/about_schools_component.html.erb

```diff
@@ -0,0 +1,53 @@
+<% if course.how_school_placements_work.present? ||
+  course.program_type == 'higher_education_programme' ||
+  course.program_type == 'scitt_programme' ||
+  course.site_statuses.map(&:site).uniq.many? %>
```

Comment:
There's a bit of duplication here with the `render?` method below in `app/components/courses/about_schools_component.rb`. 

Can we reuse that logic?

Reply:
Good spot - the duplicated code should not be in the erb file - have removed